inventory/upcloud.ini

```diff
@@ -20,3 +19,4 @@
 return_ip_addresses = True
 return_non_fqdn_names = False
 default_timeout = 300
+default_ipv_version = IPv4
```

Comment:
```suggestion
default_ip_version = IPv4
```

Reply:
Since we will be deprecating this repo I’d not make too many lifecycle management changes.